configs/field-types/multiline-text.jsx

```diff
@@ -1,13 +1,58 @@
 const React = require('react');
+const useState = require('react').useState;
 const Input = require('@material-ui/core/Input').default;
+const InputLabel = require('@material-ui/core/InputLabel').default;
+const S = require('string');
+const asField = require('informed').asField;
 
-export const Renderer = ({onChange, value}) => {
-  return <Input onChange={onChange} value={value} />;
-};
+export const Renderer = asField(({ fieldState, fieldApi, label, ...props }) => {
+  const { value } = fieldState;
+  const { setValue, setTouched } = fieldApi;
+  const { onChange, onBlur, initialValue, forwardedRef, ...rest } = props;
+  const minRows = 3;
+  const [rows, setRows] = useState(minRows);
+
+  return (
+    <React.Fragment>
+      {label && <InputLabel>{S(label).capitalize().s}</InputLabel>}
+      <Input
+        {...rest}
+        ref={forwardedRef}
+        value={!value && value !== 0 ? '' : value}
+        onChange={e => {
+          handleRowsChange(e, setRows, minRows);
+          setValue(e.target.value);
+          if (onChange) {
+            onChange(e);
+          }
+        }}
+        onBlur={e => {
+          setTouched(true);
+          if (onBlur) {
+            onBlur(e);
+          }
+        }}
+        multiline={true}
+        rows={rows}
+        style={fieldState.error ? { border: 'solid 1px red', resize: 'none' } : null}
+      />
+    </React.Fragment>
+  );
+});
+
+const handleRowsChange = (e, setRows, minRows) => {
+  const lineHeight = 19;
```

Comment:
it could be better if we get line height dynamically somehow eventually, because the styles and font will update, so the line height might change. But this will work for some time. Don't worry about it for now